app/src/main/java/com/quran/labs/androidquran/util/AudioUtils.kt

```diff
@@ -125,19 +125,7 @@ constructor(private val quranInfo: QuranInfo, private val quranFileUtils: QuranF
     if (page > totalPages || page < 0) {
       return null
     }
-    if (page < totalPages) {
-      val nextPage = page + 1
-      val nextPageSura = quranInfo.getSuraNumberFromPage(nextPage)
-      // using [page+1] as an index because we literally want the next page
-      val nextPageAyah = quranInfo.getFirstAyahOnPage(nextPage)
-
-      pageLastSura = nextPageSura
-      pageLastAyah = nextPageAyah - 1
-      if (pageLastAyah < 1) {
-        pageLastSura--
-        pageLastAyah = quranInfo.getNumberOfAyahs(pageLastSura)
-      }
-    }
```

Comment:
this was for page download amount, and there's a method for that which is also more correct than this (this code is actually wrong if you consider a case like 56:51, which was returning the sura as 57 instead of 56)